scripts/transfer_leads.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import logging
+from closeio_api import Client as CloseIO_API
+
+
+def empty_if_none(val):
+    return val if val is not None else ''
+
+parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter, description="""
+Transfering leads from one organization to another.
+
+""")
+parser.add_argument('--api-key', '-k', required=True, help='API Key')
+parser.add_argument('--target-api-key', required=True, help='Target company API Key')
+parser.add_argument('--development', '-d', action='store_true',
+                    help='Use a development (testing) server rather than production.')
+parser.add_argument('--delete', action='store_true',
+                    help='Delete transfered leads.')
+parser.add_argument('--confirmed', '-c', action='store_true',
+                    help='Without this flag, the script will do a dry run without actually updating any data.')
+parser.add_argument('query', help='Search query')
```

Comment:
this script won't work properly unless there's a `sort:date`. either put this in the help message or ensure it in code